cookiecutter/hooks.py

```diff
@@ -50,7 +52,7 @@ def find_hooks():
     return r
 
 
-def run_script(script_path, cwd='.'):
+def run_script(script_path, cwd='.', context={}):
```

Comment:
:+1: to refactoring and getting rid of the [mutable default arg](http://effbot.org/zone/default-values.htm)